src/RpcServer/Utility.cs

```diff
@@ -0,0 +1,28 @@
+using Neo.IO.Json;
+using Neo.Network.P2P.Payloads;
+using Neo.SmartContract.Native;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Neo.Plugins
+{
+    public static class Utility
```

Comment:
Helper?

Reply:
It's called Utility in RpcClient, and I've just adopted it.